core/src/block_processor.rs

```diff
@@ -80,6 +82,11 @@ impl BlockProcessor {
             .await
             .context("failed to get block")?;
 
+        // save the block in storage
+        if let Some(block_store) = &self.block_store {
```

Comment:
I think you should call it block_storage instead of block_store because it creates some confusion. block_store is already use elsewhere for other thing; See line 101 for example.

Reply:
Done